src/jobserver-win32.cc

```diff
@@ -0,0 +1,58 @@
+// Copyright 2024 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <windows.h>
+#include <cassert>
+
+#include "jobserver.h"
+#include "util.h"
+
+void Jobserver::Init() {
+  assert(sem_ == INVALID_HANDLE_VALUE);
+
+  if (!ParseJobserverAuth("sem")) {
+    return;
+  }
+
+  const char *name = jobserver_name_.c_str();
+
+  sem_ = OpenSemaphore(SEMAPHORE_ALL_ACCESS, false, name);
+  if (sem_ == INVALID_HANDLE_VALUE) {
+    Win32Fatal("OpenSemaphore");
+  }
+
+  Info("using jobserver: %s", name);
```

Comment:
Here you're it's lower case "using", should be identical to Linux. Ninja isn't consistent in the casing, choose what you prefer :)